### PR TITLE
HW 28.2: now detects userId

### PR DESCRIPTION
**Pull request by SetMyDream**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java [hunk 1]
```diff
@@ -24,9 +25,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Order> orderList = orderService.getUserOrders(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = Long.parseLong(String.valueOf(session.getAttribute(USER_ID)));
```
Comment (Vosker) [inline src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java]: Long userId = (Long) req.getSession().getAttribute(USER_ID);
You can change it like that

Comment (Vosker) [reply src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java]: and everywhere else in the order

**Review by Vosker — COMMENTED** [review]
good jobzzzz

File: src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java]: I agree with Nikita, let's use casting here

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**